core/templates/dev/head/pages/exploration_editor/EditorNavigationDirective.js

```diff
@@ -38,10 +38,14 @@ oppia.directive('editorNavigation', [function() {
         $scope.postTutorialHelpPopoverIsShown = false;
 
         $scope.$on('openPostTutorialHelpPopover', function() {
-          $scope.postTutorialHelpPopoverIsShown = true;
-          $timeout(function() {
+          if ($scope.isLargeScreen) {
```

Comment:
Hmm, on second thought (and trial), it is fine, though it might be better if we shift those variable instantiation higher up for readability's sake.